ec2/connection.py

```diff
@@ -0,0 +1,64 @@
+########
+# Copyright (c) 2014 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+# built-in package
+
+# other packages
+from boto.ec2 import EC2Connection
+from boto.ec2 import connect_to_region
+from boto.exception import EC2ResponseError
+
+# ctx packages
```

Comment:
not really ctx packages. all of these relate to the plugins-common dir.. not specifically ctx.